lib/include/mxl/flowinfo.h

```diff
@@ -75,18 +73,22 @@ extern "C"
          */
         uint32_t maxSyncBatchSizeHint;
 
-        /*
+        /**
+         * And indication, where the payload memory is located.
+         * \see mxlPayloadLocation
+         */
+        uint32_t payloadLocation;
+
+        /**
          * Device index (if payload is in device memory). -1 if on host memory.
          */
         int32_t deviceIndex;
```

Comment:
If we might use different domains for different host NUMA zones, could this also be used for that?

Reply:
You mean instead of keeping it fixed at `-1`?
I don't see a reason why not, but we need to put some thoughts into how to make that information actionable in a reasonable manner.

Reply:
👍